hack/scripts/integration-test-bootstrap.sh

```diff
@@ -274,8 +296,13 @@ function profileProw {
 }
 
 function profileStage {
+    clusterUserName=$($OC whoami)
+    $OC --as backplane-cluster-admin adm policy add-cluster-role-to-user cluster-admin $clusterUserName
```

Comment:
looks good. just add a comment here about why youre doing this.

Reply:
Added the comment.